_posts/2021-01-27-solidity-0.8.1-release-announcement.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.1 Release Announcement'
+date: '2021-01-27'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.1](https://github.com/ethereum/solidity/releases/tag/v0.8.1) adds XYZ
+
+use newer emscripten version for the build (effects?)
+
+many SMTChecker improvements
+
+abiliity to catch panic
+
+extcodesize shortcut
+
+## Important Bugfixes 
+
+
+## Notable New Features
+
+### How to select SMTChecker targets
+
+### Catch Panic
+
+The ``Panic(uint)`` revert error was introduced with 0.8.0, but there was not yet a way
+to catch and decode such an error in a ``try/catch`` statement. This has been added
+now:
+
+    try otherContract.f() { }
+    catch Panic(uint _code) {
+      if (_code == 0x01) { revert("Assertion failed"); }
+      else if (_code == 0x11) { revert ("Underflow/overflow"); }
+      // We ignore the other errors.
+    }
+
+If you do not have a ``catch Panic(uint)`` clause (nor a ``catch { ..}`` clause) in your ``try/catch`` statement,
```

Comment:
```suggestion
If you do neither have a ``catch Panic(uint)`` nor a ``catch { ..}`` clause in your ``try/catch`` statement,
```

Reply:
It's actually more complicated, because there is also `catch (bytes memory) { ... }`